### DISCUSSION TITLE
Is there any MySQL cross-schema support ?

### DISCUSSION THREAD

**Discussion by Gaibhne** [directus/directus#16306] [Archive]
Generally it is considered good practice in the MySQL world to separate concerns into different databases/schemas, and MySQL supports queries that cross database/schema boundaries (i.e. `SELECT * FROM db1.table1 JOIN db2.table2 ON db1.table1.columnA = db2.table2.columnB`). Is there any way to use a database that is set up like that in Directus ?

**Comment by rijkvanzanten** [comment]
Directus (currently) connects to a single database/schema instance, so doing cross-database joins etc isn't possible (yet).

**Comment by Gaibhne** [comment]
Those two are not mutually exclusive. MySQL supports cross schema queries despite being only 'connected' to one. For what I'm asking about, multiple database support is not actually needed (and in fact, `JOINs` as described in my question would not actually be possible with multiple database connections).

**Reply by rijkvanzanten** [reply]
Lemme rephrase my answer as:

Directus (currently) connects to / manages / queries against a single database/schema instance, so doing cross-database joins etc isn't possible (yet).

**Comment by Gaibhne** [comment]
Ah okay, I just wanted to make sure we didn't misunderstand each other. For what it's worth, this feature would definitely be something to make Directus stand out, as it is (unfortunately for me) one not supported by any of the competitors like NocoDB or Motor Admin either.
